pkg/controller/node/nodecontroller.go

```diff
@@ -343,6 +386,17 @@ func forcefullyDeletePod(c client.Interface, pod *api.Pod) {
 	}
 }
 
+// releaseCIDR does translating CIDR back to CIDR index and insert this index
+// back to availableCIDRs set
+func (nc *NodeController) releaseCIDR(freeCIDR string) {
+	CIDRsArray := strings.Split(freeCIDR, ".")
+	cidrIP := nc.clusterCIDR.IP.To4()
+	CIDRsIndexOne, _ := strconv.Atoi(CIDRsArray[1])
+	CIDRsIndexTwo, _ := strconv.Atoi(CIDRsArray[2])
+	release := (CIDRsIndexOne-int(cidrIP[1]))*256 + CIDRsIndexTwo - int(cidrIP[2])
```

Comment:
It will not happen since translateCIDRs only translate index of CIDR into real CIDR. The index overflow(means no more CIDR in the system) checking is in generateNewAvailabeCIDRs(), and real CIDR overflow(means the CIDR is not valid CIDR, the example of not valid is 10.2.257.0/16) checking is in translateCIDRs.